### ISSUE TITLE
Error compiling use Browserify

### ISSUE THREAD

**Issue by emhagman** [BranchMetrics/Web-SDK#36]
```
Error: Cannot find module 'cordova/exec' from '/Users/erich/projects/drafted-server/client/node_modules/branch-sdk/dist/web'
    at /Users/erich/projects/drafted-server/client/node_modules/browserify/node_modules/resolve/lib/async.js:46:17
    at process (/Users/erich/projects/drafted-server/client/node_modules/browserify/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Users/erich/projects/drafted-server/client/node_modules/browserify/node_modules/resolve/lib/async.js:188:17)
    at load (/Users/erich/projects/drafted-server/client/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/erich/projects/drafted-server/client/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /Users/erich/projects/drafted-server/client/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:99:15)
```

Removing the line that checks for `config.CORDOVA_BUILD` fixes it even though the `config.CORDOVA_BUILD` is set to `!1` in the `config` var for the `/dist/web/build.js` file.

**Comment by dmitrig01** [comment]
Agh, you're right. Let me try and figure that out.

**Comment by dmitrig01** [comment]
Are you using bower?

**Comment by emhagman** [comment]
Nope, using npm

**Comment by dmitrig01** [comment]
Ok, I have a fix. Just runnign a few tests, and hopefully we can release later this morning

**Comment by emhagman** [comment]
Thanks! Appreciate it.

**Comment by dmitrig01** [comment]
https://github.com/BranchMetrics/Web-SDK/pull/37 :-)

**Comment by dmitrig01** [comment]
All right - i just published 1.4.1. Let me know if it works for you.

**Comment by dmitrig01** [comment]
gonna go ahead and close this. please reopen if you ahve any more issues
